doc/tech/usecases.md

```diff
@@ -11,3 +11,17 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+
+2-Objective :
+
+
+Based on CNTT’s Project goals & Purpose was formulated is to develop a robust infrastructure model and define a limited discrete set of architectures built on that model that can be tested and validated for use across the entire member community. 
```

Comment:
Suggest change:
"Based on CNTT’s Project goals & Purpose was formulated is to develop a robust infrastructure model and define a limited discrete set of architectures ..." --> "CNTT’s goal and purpose is to develop a robust cloud infrastructure model and define a limited set of discrete architectures ..."